database/insertData/laos/02_template_orgRegistration.js

```diff
@@ -0,0 +1,1003 @@
+                      parameters: {
+                        label: "Phone"
+                        description: "TO-DO: Pre-populate"
+                        maxLength: 50
+                      }
+                      validation: {
+                        operator: "REGEX"
+                        children: [
+                          {
+                            operator: "objectProperties"
+                            children: ["responses.thisResponse"]
+                          }
+                          "^[0-9()\\\\-\+]*$"
+                        ]
+                      }
+                      validationMessage: "Not a valid phone number"
+                    }
+                    {
+                      code: "ownerEmail"
+                      index: 120
+                      title: "Company Owner Email"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      parameters: {
+                        label: "Email"
+                        description: "TO-DO: Pre-populate"
+                        maxLength: 150
+                      }
+                    }
+                    {
+                      code: "ownerFax"
+                      index: 130
+                      title: "Company Owner Fax"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      isRequired: false
+                      parameters: {
+                        label: "Fax number"
+                        description: "(if applicable)"
+                        maxLength: 150
+                      }
+                      validation: {
+                        operator: "REGEX"
+                        children: [
+                          {
+                            operator: "objectProperties"
+                            children: ["responses.thisResponse"]
+                          }
+                          "^[0-9()\\\\-\+]*$"
+                        ]
+                      }
+                      validationMessage: "Not a valid fax number"
+                    }
+                  ]
+                }
+              }
+              {
+                code: "S2"
+                title: "Professional Details"
+                index: 1
+                templateElementsUsingId: {
+                  create: [
+                    {
+                      code: "S2Intro"
+                      index: 10
+                      title: "Intro Section 2"
+                      elementTypePluginCode: "textInfo"
+                      category: INFORMATION
+                      helpText: "The following questions are regarding the professional qualifications and experience of the **individual** who will primarily act on the company's behalf"
+                      parameters: {
+                        title: "## Professional details"
+                        text: "To-do: show Users name here with look-up"
```

Comment:
Not sure if I understood this TODO

Reply:
I just meant the the TextInfo field should show:
### Professional details
#### Applicant's Name

to indicate who we're filling out the info for.

I'll do this shortly

Reply:
Done now as part of 6b864f08

Looks like:
![Screen Shot 2021-06-10 at 2 07 20 PM](https://user-images.githubusercontent.com/5456533/121453396-36e3c780-c9f5-11eb-977f-63a9f0de647a.png)